packages/vdom-extension/package.json

```diff
@@ -34,12 +34,12 @@
     "@nteract/transform-vdom": "^1.1.1",
     "@phosphor/messaging": "^1.2.2",
     "@phosphor/widgets": "^1.6.0",
-    "react": "~16.4.2",
-    "react-dom": "~16.4.2"
+    "react": "~16.8.3",
```

Comment:
This will widen the gap between the already-incompatible dependency via `@nteract/transform-vdom": "^1.1.1`, see https://github.com/jupyterlab/jupyterlab/issues/5986#issuecomment-471447740

Reply:
Thanks for pointing this out @javabrett. #5670 should fix that problem.